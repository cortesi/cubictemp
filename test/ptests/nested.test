<!--(block foo)-->
	<!--(for this in range(5))
        @!this!@
	(end)-->
<!--(end)-->
@!foo!@

<!--(block bar)-->
	<!--(for this in range(2))
        @!this!@
		<!--(for this in range(2))
            @!this!@
		(end)-->
	(end)-->
<!--(end)-->
@!bar!@

<!--(for this in range(2))
	<!--(block moo)
        bar
	(end)-->
    @!moo!@
(end)-->

<!--(block outer)
    barg
(end)-->
<!--(block one)
    @!outer!@
(end)-->
@!one!@

<!--(for this in range(2))
	<!--(for that in range(2))
        @!this!@@!that!@
	(end)-->
(end)-->
