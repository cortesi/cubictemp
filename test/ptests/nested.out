0
1
2
3
4

0
0
1
1
0
1

bar
bar

barg

00
01
10
11
